services/QuillLMS/engines/evidence/lib/evidence/evidence/change_log.rb

```diff
@@ -63,12 +63,30 @@ def activity_versions
       Evidence.change_log_class.where(changed_record_id: id, changed_attribute: 'version', changed_record_type: 'Evidence::Activity').map do |row|
         {
           note: row.explanation,
-          updated_at: row.updated_at,
-          new_value: row.new_value
+          created_at: row.created_at,
+          new_value: row.new_value,
+          session_count: session_count(row, id)
         }
       end
     end
 
+    def session_count(change_log, activity_id)
+      start_date = change_log.created_at
+      end_date = Time.current
+
+      if change_log.previous_value != "0"
```

Comment:
Should this be "1" now that 1 is the default?  Also, given that 1 is magic number in many pieces of this commit, I would extract it out as a constant:  DEFAULT_VERSION = 1

Then you have: `change_log.previous_values != DEFAULT_VERSION.to_s`

Reply:
so this should still be 0-- we initially did not have any version `ChangeLog` entries for when an Evidence activity was created. Now that we are starting at version one, version ChangeLog entries were backfilled for activity creation with `previous_value` being 0 and `new_value` being 1, so an activity that's on the first version won't step into this if statement.

good call on the default_version constant, I'll add that